src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs

```diff
@@ -370,5 +370,187 @@ public void CreateStyle_IsBold_IsItalic()
 
             Assert.IsTrue(fontStyle == XFontStyle.BoldItalic, "fontStyle should be BoldItalic");
         }
+
+        #region Font name mapping tests
```

Comment:
Good looking tests.  For maintenance and tracking, these should go into a new Test Class for XFontTests under the Drawing Folder.